Makefile

```diff
@@ -244,13 +244,13 @@ test-benchmark: ## Run the benchmark tests -- Note that this can only be ran for
 e2e: namespace tear-down operator-sdk image-to-cluster openshift-user deploy-crds ## Run the end-to-end tests
 	@echo "WARNING: This will temporarily modify deploy/operator.yaml"
 	@echo "Replacing workload references in deploy/operator.yaml"
-	@$(SED) 's%$(IMAGE_REPO)/$(RELATED_IMAGE_OPENSCAP_NAME):$(OPENSCAP_DEFAULT_IMAGE_TAG)%$(RELATED_IMAGE_OPENSCAP_PATH):$(RELATED_IMAGE_OPENSCAP_TAG)%' deploy/operator.yaml
+	@$(SED) 's%$(IMAGE_REPO)/$(RELATED_IMAGE_OPENSCAP_NAME):$(OPENSCAP_DEFAULT_IMAGE_TAG)%$(RELATED_IMAGE_OPENSCAP_PATH):$(TAG)%' deploy/operator.yaml
```

Comment:
Instead of switching to TAG here and in the `cluster-image-push` target, let's change the `openscap-image` target to use RELATED_IMAGE_OPENSCAP_TAG. 

We don't update the openscap image very frequently and when we rebuild it will still be 1.3.4, so we can keep using the existing tag.

Reply:
Sounds Good!